src/System.Private.CoreLib/shared/System/Globalization/CultureData.cs

```diff
@@ -974,8 +954,10 @@ internal string SENGDISPLAYNAME
             }
         }
 
-        // Native pretty name for this locale (ie: Deutsch (Deutschland))
-        internal string SNATIVEDISPLAYNAME
```

Comment:
Need `CultureData.Windows.cs` updated to use the new names too.

Reply:
Fixed